packages/admin-next/dashboard/src/v2-routes/shipping/service-zone-create/components/create-service-zone-form/create-service-zone-form.tsx

```diff
@@ -0,0 +1,285 @@
+
+                <div>
+                  <Text weight="plus">
+                    {t("shipping.serviceZone.create.subtitle")}
+                  </Text>
+                  <Text className="text-ui-fg-subtle mb-8 mt-2">
+                    {t("shipping.serviceZone.create.description")}
+                  </Text>
+                </div>
+
+                <div className="flex max-w-[340px] flex-col gap-y-6">
+                  <Form.Field
+                    control={form.control}
+                    name="name"
+                    render={({ field }) => {
+                      return (
+                        <Form.Item>
+                          <Form.Label>
+                            {t("shipping.serviceZone.create.zoneName")}
+                          </Form.Label>
+                          <Form.Control>
+                            <Input {...field} />
+                          </Form.Control>
+                          <Form.ErrorMessage />
+                        </Form.Item>
+                      )
+                    }}
+                  />
+                </div>
+              </div>
+
+              {/*AREAS*/}
+              <div className="container flex items-center justify-between py-8">
+                <div>
+                  <Text weight="plus">
+                    {t("shipping.serviceZone.areas.title")}
+                  </Text>
+                  <Text className="text-ui-fg-subtle mt-2">
+                    {t("shipping.serviceZone.areas.description")}
+                  </Text>
+                </div>
+                <Button
+                  onClick={() => setOpen(true)}
+                  variant="secondary"
+                  type="button"
+                >
+                  {t("shipping.serviceZone.areas.manage")}
+                </Button>
+              </div>
+              {!!selectedCountries.length && (
+                <div className="flex items-center gap-4">
+                  <div className="grow-0 rounded-lg border">
+                    <div className="bg-ui-bg-field m-1 rounded-md p-2">
+                      <Map className="text-ui-fg-subtle" />
+                    </div>
+                  </div>
+                  <ListSummary
+                    inline
+                    list={selectedCountries.map((c) => c.display_name)}
+                  />
+                </div>
+              )}
+            </SplitView.Content>
+            <SplitView.Drawer>
+              <div className="flex size-full flex-col overflow-hidden">
+                <DataTable
+                  table={table}
+                  columns={columns}
+                  pageSize={PAGE_SIZE}
+                  count={count}
+                  search
+                  pagination
+                  layout="fill"
+                  orderBy={["name", "code"]}
+                  queryObject={raw}
+                  prefix={PREFIX}
+                />
+              </div>
+            </SplitView.Drawer>
```

Comment:
We should add a footer with an "Add" and "Cancel" button, and wait with adding the countries until the user clicks "Add".